conans/test/functional/cppstd/default_cppstd_test.py

```diff
@@ -34,14 +33,15 @@ class Library(ConanFile):
 
             def configure(self):
                 cppstd = self.settings.get_safe("cppstd")
-                self.output.info(">>>> settings: {{}}".format(self.settings.fields))
-                self.output.info(">>>> cppstd: {{}}".format(cppstd))
+                compiler_cppstd = self.settings.get_safe("compiler.cppstd")
```

Comment:
Do we want to alter this test? Or keep it to ensure we don't break anything and do another one for the new subsetting?

Reply:
I've minimized changes in this file, now I think it is clear that the modifications (additions) don't affect existing behavior. Tell me what you think, thanks!